application-tour-ui/src/main/resources/TourCode/StepSheet.xml

```diff
@@ -238,6 +239,7 @@ require(['jquery'], function ($) {
                   targetPage  : (targetPage? targetPage: ''),
                   targetAction: (targetAction? targetAction: ''),
                   queryString : (queryString? queryString: ''),
+                  reflex      : (reflex? reflex: 0),
```

Comment:
That doesn't make it right. It may not be wrong but we need to understand why you have to return ``0`` when ``backdrop`` and ``reflex`` are ``false``? In any case, you shouldn't copy code that you don't fully understand. Maybe that line is a bug and you propagate it to ``reflex``.